src/classes/HH_NameSpec_TEST.cls

```diff
@@ -396,6 +402,51 @@ private with sharing class HH_NameSpec_TEST {
     /*********************************************************************************************************
     @description
     operation:
+        Test Household Name, and Formal/Informal greetings when Contact API names are case insensitive
+    verify:
+        HH Name, Formal Greeting, and Informal Greeting are correct
+    **********************************************************************************************************/            
+    static testMethod void testHHNamingWhenContactApiNamesAreCaseInsensitive() {
```

Comment:
so here's an example of a shorter name that might still work.  since this is in the HH_NameSpec_TEST class, we already know it's about HHNaming, so we don't really need to include "testHHNamingWhen".  I probably would have called this something like testNameFormatsCaseInsensitive()